src/backend/crossterm.rs

```diff
@@ -274,6 +274,32 @@ impl From<Color> for CColor {
     }
 }
 
+impl From<CColor> for Color {
+    fn from(value: CColor) -> Self {
+        match value {
+            CColor::Reset => Self::Reset,
```

Comment:
nit: is it worth making these in the same order as above (it's that way because it aligns with the numeric index of the colors)?

Reply:
Sure, I put them in the order of the enum originally